chicago/views.py

```diff
@@ -332,6 +332,43 @@ def get_context_data(self, **kwargs):
             .order_by("-last_action")[:10]
         )
 
+        attendance = []
+        events = []
+        # get all events across all memberships tenures
+        for membership in person.current_memberships.all():
+            events.extend(
+                ChicagoEvent.objects.filter(
+                    participants__entity_type="organization",
+                    status="passed",
+                    participants__name=membership.organization.name,
```

Comment:
this would be better as `participants__organization_id=memberships.organization.id`